images/kubekins-e2e/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+reviewers:
+- amwat
+- spiffxp
+- BenTheElder
+approvers:
+- amwat
+- spiffxp
```

Comment:
oh boy, now where is that elixir of knowledge that grants me all the history behind what's in here

Reply:
nobody actually has this :-)
if you ask sen and I about it and dig through git you might figure it out.

for this image we should just generally avoid major changes.

the krte image is simpler and I'm attempting to leave comments for everything.
we should plan to move away from the bootstrap cruft anyhow :sweat_smile: